google-cloud-storage/lib/google/cloud/storage/hmac_key.rb

```diff
@@ -164,7 +164,8 @@ def active?
         ##
         # Updates the state of the HMAC key to `ACTIVE`.
         #
-        # @return [Boolean] Returns `true` if the HMAC key was updated.
+        # @return [Google::Cloud::Storage::HmacKey] Returns the HMAC key for
+        #   chaining.
```

Comment:
What do you think about changing this to "chaining methods" or "method chaining"?

Reply:
Updated.